INSTALL

```diff
@@ -0,0 +1,213 @@
+Prerequisites:
+==============
+1. Python Git: sudo apt-get install python-git
+2. Google repo: https://source.android.com/source/downloading
+3. Checked with Python v 2.7.12, but other should also work
+
+About:
+======
+Gen3-test product is based on Renesas's BSP and Xen hypervisor. It contains a set of features required to have
+both Display and GPU support in several domains at the same time (device passthrough, PVDRM, VGPU, etc).
+
+There are three domains which are running on top of Xen:
+1. Generic machine independent control domain named "domain 0". This "initramfs" based domain is responsible
+for managing VM only (create/destroy/reboot guest domains). There is no HW assigned to this domain.
+2. Machine dependent driver domain named "domain D". Almost all HW are assigned to this domain.
+As this domain is not 1:1 mapped, Xen controls IPMMU to do a proper address translations (IPA to PA) for DMA masters.
+It also contains different PV backends running inside it in order to provide guests domains (without HW assigned)
+with different services such as Audio, Display, Net, Block, etc.
+3. Machine dependent simple guest domain named "domain U". It has different PV frontends running and
+doesn't have any HW assigned except GPU. The latter are shared between "domain D" and "domain U" in current setup.
+
+Build:
+======
+Our new build system uses set of additional meta layers and tools.
+1. Please, clone the following build scripts, master branch:
+
+git clone https://github.com/xen-troops/build-scripts.git
+cd build-scripts
+
+2. In the build-scripts directory you will find a sample configuration
+file xt-build-server.cfg:
+
+cp xt-build-server.cfg xt-gen3-test.cfg
+
+3. Edit it to fit your environment: the variables we need to change
+are under [path] section. Please set:
+ - workspace_base_dir: change it to point to the place where the build will happen
+ - workspace_storage_base_dir: change it to where downloads and other files will be put
+
+For example,
+
+workspace_base_dir = /tmp/workspace_base
```

Comment:
never put /tmp in examples for build target: people will use it as is and will loose all after reboot...